development/cpp/configuring_an_ide/visual_studio_code.rst

```diff
@@ -83,13 +83,13 @@ To run and debug the project you need to create a new configuration in the ``lau
 - Select **C++ (GDB/LLDB)**. There may be another platform specific option here. If selected,
   adjust the configuration example provided accordingly.
 - Within the ``launch.json`` file find the ``"configurations"`` array and add a new section to it:
-
+.. note:: Due to sporadic performance issues, it is recommended to use LLDB over GDB on Unix-based systems. Make sure that the `CodeLLDB extension <https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb>`_ is installed.
```

Comment:
This needs to have an empty line before and after, and we also keep the text lines to a wrapping length.

```suggestion

.. note::

    Due to sporadic performance issues, it is recommended to use LLDB over GDB on Unix-based systems.
    Make sure that the `CodeLLDB extension <https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb>`_
    is installed.

```

I would also probably move it below the config examples, so that the flow is not broken.

Reply:
Should I squash it?

Reply:
You didn't move it below yet 🙃 But sure, squash it if you can.